### ISSUE TITLE
Welcome to tweets.opml.org

### ISSUE THREAD

**Issue by scripting** [scripting/drummerSupport#130]
A pointer to the announcement will go here momentarily.

**Comment by mistersugar** [comment]
This worked for me. A new tweet showed up in the outline as expected.

**Comment by scripting** [comment]
Thanks for the quick test @mistersugar. 

I should have mentioned that the outlines are public, on GitHub, so anyone can see what's developing there. 

https://github.com/scripting/tweets.opml.org/tree/main/users/mistersugar

I love the idea of using GH as a public storage system. ;-)

**Comment by am1t** [comment]
Nice, it worked as expected for me. This should be a good option to use Twitter to capture quick thoughts. _And threads work too_, nice! 

I do have a couple of questions, please excuse if they don't fit your thought process yet.

1. Given this is a public outline, will it be (in future) indexed to be searchable in Daytona?
2. Tweets with links are not fetched at this point. Is that correct? I did not see a reference in the announcement post.

All said, an interesting change, Dave!

**Comment by scripting** [comment]
@am1t — glad it worked. I’ll have comments tomorrow (in my time zone).

**Comment by akaKenSmith** [comment]
Looks good. I signed in and after posting a tweet saw it pop up soon in Drummer.

One small thing:

Somewhere in the process special characters -- the ampersand and curly quotation marks -- were translated into code, such as "&" changing to "&amp"... 

These changes of characters to code were visible in the Github outline and in the Drummer instant outline.

**Comment by akaKenSmith** [comment]
Now it looks like the new tweets outline has overwritten the blog outline that I had open at the same time in Drummer. I have a pretty recent backup, so I'm not especially concerned at this point. I'll see about dragging the backup copy of the blog file into place and report back.

**Comment by scripting** [comment]
@am1t -- glad you like tweets.opml.org. :-)

1. [daytona.ping](http://docserver.scripting.com/?verb=daytona.ping) will index any public OPML file you want to include in your index. You don't have to include the _collection_ param because it defaults to drummeruser, which is what you want for this call. 

2. It's supposed to work with tweets-with-links, but now that you mention it, that doesn't appear to be working. I'll have a look.

**Comment by am1t** [comment]
I can confirm I have my tweets indexed too now. This is so brilliant. I want all my writing indexed and searchable in Daytona now. Thank you, Dave 👍🏽

**Comment by scotthansonde** [comment]
@troutcolor Try posting a tweet with no link included. The system seems for now to ignore tweets that have links.

**Comment by troutcolor** [comment]
> @troutcolor Try posting a tweet with no link included. The system seems for now to ignore tweets that have links.

@scotthansonde that did the trick. I’d guess nearly all of my tweets have a link in them. Thanks.

**Comment by scripting** [comment]
Just posted a [change note](http://scripting.com/drummer/blog/2021/12/18/162625.html?title=tweetsopmlorgProgress) about tweets.opml.org.

* The new software is installed on the server.
* Links should now be working.
* There was a problem with RTs, which it ignores, but when it encountered an RT, it would fail.
* Still have some more work to do on the UI. 

If you want to help test, post a few tweets and see if they show up in your OPML file on GitHub.
Knock wood, I think we're good to go now.

**Comment by am1t** [comment]
I tested with a few tweets. They do show up as expected. Even those with links (though the links are shortened, with t.co domain). Nice!

**Comment by troutcolor** [comment]
Briefly tested and working. I also note that emoji are now displaying my tweets opml in drummer :-)

**Comment by scripting** [comment]
### A different URL for your OPML file

This is important. 

My OPML file is now located here.

* http://tweets.opml.org/davewiner/tweets.opml

And yours is located in a similar location.

I have modified <a href="http://tweets.opml.org/">the app</a> so that it displays this URL on the home page. 

There are two reasons this is a better URL.

1. It's an instant outline, when you open it in Drummer, it will subscribe to updates. 
2. It's possible in the future to redirect it elsewhere, there was no way to do that with the GitHub url.

The server will continue to update the GitHub file and keep the archive on GitHub. 

This is a better one to share and to subscribe to.

**Comment by am1t** [comment]
> A different URL for your OPML file

This one looks and sounds a lot better than the raw GitHub one. Great! 👍🏽

**Comment by scripting** [comment]
@am1t -- yeah that was a mistake -- this is a much better url.

also I have code that converts the t.co link to its real link, but that depends on t.co cooperating, but it's unreliable.

for now it's simpler just not to try.

**Comment by am1t** [comment]
> for now it's simpler just not to try.

Yeah, just wanted to report. I think it's OK to not worry about the shortened URL, for now.

**Comment by scripting** [comment]
@am1t, @troutcolor -- thanks for the quick confirmation the new software works.

Now I can relax while eating my lunch and start thinking about the next thing I am going to try to finish today or tomorrow.

As always I tend to juggle a few balls concurrently. 😀

**Comment by akaKenSmith** [comment]
New tweets showed up on my github outline just now, yes.

**Comment by scripting** [comment]
I fleshed out the [readme file](https://github.com/scripting/tweets.opml.org/tree/main#readme) on the tweets.opml.org site.

**Comment by scripting** [comment]
We were doing the archives incorrectly. 
The way it should work is a folder for every year, and inside the folder, the months are 01.opml, 02.opml and so on.
I fixed the app so it now does it correctly, and deleted the archive folder for each user.
An example of a correct structure, look at the <a href="https://github.com/scripting/tweets.opml.org/tree/main/users/cluelessnewbie">archive</a> for cluelessnewbie. 
The next time you post a tweet, it will start creating a correctly structured archive.

**Comment by scripting** [comment]
How to follow my Tweets in Drummer. 

http://scripting.com/2021/12/19/152625.html?title=followingMyTweetsInDrummer

**Comment by cbates-** [comment]
When I try to log in to tweets.opml.org I end up this :
![image](https://user-images.githubusercontent.com/1412601/146964671-ab2c9cec-85e8-4650-9089-ea327b3cedbc.png)

That's after using my email address for a 2nd time trying to log in to twitter.

**Comment by scripting** [comment]
@cbates- that's not my dialog. what did you do, the actual steps.

**Comment by cbates-** [comment]
1. Point the browser (Firefox, Win10) at tweets.opml.org.
2. _Sign on here_ -> _Sign on Twitter..._
![image](https://user-images.githubusercontent.com/1412601/146968467-ae56f27f-6ee9-4d91-92de-ef9e61c48acf.png)
At this point (when I tried it in Brave browser) I got the message in the previous screenshot.

It works in Firefox, found my url on the tweets.opml.org _Menu_.
Thanks --

**Comment by scripting** [comment]
@cbates - I figured it was something like that. Not sure what it means.
